test/conformance/queue/urQueueCreateWithNativeHandle.cpp

```diff
@@ -30,3 +30,58 @@ TEST_P(urQueueCreateWithNativeHandleTest, Success) {
     ASSERT_EQ(q_context, context);
     ASSERT_SUCCESS(urQueueRelease(q));
 }
+
+TEST_P(urQueueCreateWithNativeHandleTest, InvalidNullHandle) {
+    ur_native_handle_t native_handle = 0;
+    {
+        UUR_ASSERT_SUCCESS_OR_UNSUPPORTED(
+            urQueueGetNativeHandle(queue, nullptr, &native_handle));
+    }
+
+    ur_queue_handle_t q = nullptr;
+    ASSERT_EQ(urQueueCreateWithNativeHandle(native_handle, nullptr, device,
+                                            nullptr, &q),
+              UR_RESULT_ERROR_INVALID_NULL_HANDLE);
+}
+
+TEST_P(urQueueCreateWithNativeHandleTest, InvalidNullPointer) {
+    ur_native_handle_t native_handle = 0;
+    {
+        UUR_ASSERT_SUCCESS_OR_UNSUPPORTED(
+            urQueueGetNativeHandle(queue, nullptr, &native_handle));
+    }
+
+    ASSERT_EQ(urQueueCreateWithNativeHandle(native_handle, context, device,
+                                            nullptr, nullptr),
+              UR_RESULT_ERROR_INVALID_NULL_POINTER);
+}
+
+TEST_P(urQueueCreateWithNativeHandleTest, SuccessWithOwnedNativeHandle) {
```

Comment:
would you mind leaving this and `SuccessWithUnOwnedNativeHandle` out? I'm updating how we test native properties along with the spec change in https://github.com/oneapi-src/unified-runtime/pull/2193

Reply:
No worries! I've removed them now